packages/dashboard-backend/src/devfileSchemas/2.2.0/devfile.json

```diff
@@ -0,0 +1,2048 @@
+{
+  "description": "Devfile describes the structure of a cloud-native devworkspace and development environment.",
+  "type": "object",
+  "title": "Devfile schema - Version 2.2.0-alpha",
```

Comment:
I think this schema is not actual anymore, 2.2.0 has been released - https://raw.githubusercontent.com/devfile/api/2.2.x/schemas/latest/ide-targeted/devfile.json